index.yaml

```diff
@@ -19,59 +19,67 @@ entries:
     version: 0.1.0
   cloudify-manager-worker:
   - apiVersion: v2
-    created: "2022-06-07T14:35:14.956251+03:00"
+    created: "2022-07-18T16:56:32.837396+03:00"
+    description: Cloudify manager worker
+    digest: 27529bf446f1fe22d27efda4774907d42d0b4b6c42b1f2da2810d04d643f2731
+    name: cloudify-manager-worker
+    urls:
+    - https://cloudify-cosmo.github.io/cloudify-helm/packages/cloudify-manager-worker-0.2.3.tgz
+    version: 0.2.3
+  - apiVersion: v2
+    created: "2022-07-18T16:56:32.836426+03:00"
     description: Cloudify manager worker
     digest: 73bf0874a78a2b7100d84da2c2f1193ddd3a41922569f8d882b6ca922741212e
     name: cloudify-manager-worker
     urls:
     - https://cloudify-cosmo.github.io/cloudify-helm/packages/cloudify-manager-worker-0.2.2.tgz
     version: 0.2.2
   - apiVersion: v2
-    created: "2022-06-07T14:35:14.955415+03:00"
+    created: "2022-07-18T16:56:32.834581+03:00"
     description: Cloudify manager worker
     digest: c9b0ff87fb5066fadd7566e319539c39f9ffef299f080f7f3344494c1d600b69
     name: cloudify-manager-worker
     urls:
     - https://cloudify-cosmo.github.io/cloudify-helm/packages/cloudify-manager-worker-0.2.1.tgz
     version: 0.2.1
   - apiVersion: v2
-    created: "2022-06-07T14:35:14.954038+03:00"
+    created: "2022-07-18T16:56:32.833035+03:00"
     description: Cloudify manager worker
     digest: 80913e82632feb31425f15cadab587f1937db4662f403eee86f059ed5de06f38
     name: cloudify-manager-worker
     urls:
     - https://cloudify-cosmo.github.io/cloudify-helm/packages/cloudify-manager-worker-0.2.0.tgz
     version: 0.2.0
   - apiVersion: v2
-    created: "2022-06-07T14:35:14.94712+03:00"
+    created: "2022-07-18T16:56:32.827078+03:00"
     description: Cloudify manager worker
     digest: 52469c8723a55c4bdcc6bc1e4a54c81f56f77fca8801d1c08bdf483351b5aef8
     name: cloudify-manager-worker
     urls:
     - https://cloudify-cosmo.github.io/cloudify-helm/packages/cloudify-manager-worker-0.1.9.tgz
     version: 0.1.9
   - apiVersion: v2
-    created: "2022-06-07T14:35:14.926559+03:00"
+    created: "2022-07-18T16:56:32.825356+03:00"
```

Comment:
Please check if we need `created` and `digest` property if not let's remove them from all the files